posthog/hogql_queries/insights/test/test_retention_query_runner.py

```diff
@@ -1055,6 +1055,56 @@ def test_all_events(self):
             ),
         )
 
+    def test_all_events_with_minimum_occurrences(self):
```

Comment:
can you add more tests for this change
- with breakdowns
- with cumulative retention
- with person property filter